modular_splurt/code/modules/jobs/job_types/_job_alt_titles.dm

```diff
@@ -166,9 +223,10 @@
 	var/list/extra_titles = list(
 		"Hydroponicist",
 		"Farmer",
-		"Beekeeper",
+		"Apiarist",
```

Comment:
Idk why the change, aren't them both the same?

Reply:
yes, but apiarist is fancy